Docs/Book/Install.md

```diff
@@ -67,6 +67,38 @@ Once "make" and "make install" completed successfully, use the following command
 This is required due to the [System Integrity Protection SIP](https://en.wikipedia.org/wiki/System_Integrity_Protection)
 introduced in more recent macOS versions.
 
+#### Linux x86_64: Python 3 environment
+
+The following commands will create a development environment for Linux x86_64 and Python 3.
+
+Start by downloading the latest anaconda installer from [Anaconda](https://www.anaconda.com/download/#linux) and install it. Then, install the required packages:
+
+	bash Anaconda3-5.2.0-x86_64.sh
+    conda install -y cmake cairo pillow eigen pkg-config
+    conda install -y boost-cpp boost py-boost
+
+Numpy and matplotlib are already part of the base installation of anaconda. Due to the latest boost libraries being currently built with a GLIBC version higher than the default in anaconda, we need to update to a more recent version:
+
+	conda install -y gxx_linux-64
+
+After that, numpy's include path must be fixed to allow RDKit to find its headers in the proper location. For that, from your anaconda root directory (usually ~/anaconda3/), run:
+
+	ln -s ./lib/python3.6/site-packages/numpy/core/include/numpy ./include/
```

Comment:
Can’t you accomplish the same thing via a command line argument to cmake? This would save the user from having to modify their conda installation

Reply:
I think `-D PYTHON_NUMPY_INCLUDE_PATH="$SP_DIR/numpy/core/include"` should work (adjusted to point to the anaconda site-packages directory)

Reply:
This was a personal preference. I prefer to have the numpy headers available in the include dir.

As @mcs07 points out, setting PYTHON_NUMPY_INCLUDE_PATH would be the alternative, although on my system I do not see a $SP_DIR env variable (according to documentation, it seems to be set by 'build', which we are not using here, to build anaconda packages), so the path would need to be set to something like "$CONDA_PREFIX/lib/python3.6/site-packages/numpy/core/include".